subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -472,9 +472,9 @@ class SubscriptionService(
                         ),
                         status = row.get("status", String::class.java)?.let { NotificationParams.StatusType.valueOf(it) },
                         timesSent = row.get("times_sent", Integer::class.java)!!.toInt(),
-                        lastNotification = row.get("last_notification", OffsetDateTime::class.java),
-                        lastFailure = row.get("last_failure", OffsetDateTime::class.java),
-                        lastSuccess = row.get("last_success", OffsetDateTime::class.java)
+                        lastNotification = row.get("last_notification", ZonedDateTime::class.java),
```

Comment:
This doesn't need to be converted to UTC either ?

Reply:
yes, fixed.